src/dotnet/commands/dotnet-install/InstallCommandParser.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.CommandLine;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal static class InstallCommandParser
+    {
+        public static Command Install()
+        {
+            return Create.Command(
+                "install", "",
+                Accept.NoArguments(), CommonOptions.HelpOption(), InstallTool());
+        }
+
+        private static Command InstallTool()
+        {
+            return Create.Command("tool",
+                "Install tool",
+                Accept.ExactlyOneArgument(o => "packageId")
```

Comment:
@KathleenDollard all the name and description are customer facing string. They are shown in --help

Reply:
Version of the package in NuGet (no dupe package)
NuGet configuration file
Target Framework Moniker (TFM) for the tools to install

Reply:
For the target framework, we should use the same text (or a text in the same vein) used everywhere else that framework is a param. For instance, for build, the help text is `Target framework to publish for. The target framework has to be specified in the project file.`

Reply:
Changed to "Target framework to publish for. The target framework has to be specified in the project file."